source/core/serial_solid.cc

```diff
@@ -276,34 +276,32 @@ SerialSolid<dim, spacedim>::get_triangulation()
 
 template <>
 void
-SerialSolid<1, 2>::rotate_grid(double /*angle*/, int /*axis*/)
+SerialSolid<1, 2>::rotate_grid(double /*angle*/, Tensor<1, 3> /*axis*/)
 {
   // Not implemented right now
   throw(std::runtime_error("This is currently not implemented"));
 }
 
 template <>
 void
-SerialSolid<2, 2>::rotate_grid(double /*angle*/, int /*axis*/)
+SerialSolid<2, 2>::rotate_grid(double /*angle*/, Tensor<1, 3> /*axis*/)
 {
   // Not implemented right now
   throw(std::runtime_error("This is currently not implemented"));
 }
 template <>
 void
-SerialSolid<3, 3>::rotate_grid(double /*angle*/, int /*axis*/)
+SerialSolid<3, 3>::rotate_grid(double /*angle*/, Tensor<1, 3> /*axis*/)
 {
   // Not implemented right now
   throw(std::runtime_error("This is currently not implemented"));
 }
```

Comment:
I think you can implement the <3,3> version right now also. It will work in the same way as the <2,3> version